tests/index.js

```diff
@@ -206,15 +206,15 @@ describe('server', function() {
       })
     });
 
-    it('facebook should 302 on random url for facebook', function(done) {
+    it('facebook should redirect to best treatment when sent a share URL with no abver value', function(done) {
       request.get({
         'url': baseUrl + '/r/0/'+URL_AB_NOHTTP+'/NO_testshare_URL_HERE?abid=1',
         'followRedirect': false,
         'headers': {
           'User-Agent': "facebookexternalhit/1.1 (+http://www.facebook.com/externalhit_uatext.php)"
         }
       }, function(err, response, body) {
-        expect(response.statusCode).to.equal(302);
+        expect(response.statusCode).to.equal(200);
```

Comment:
yeah, maybe test that the response content/body?  has the right title (or at least *a* title or og tag in content)